Document/0x04a-Android.md

```diff
@@ -160,6 +160,66 @@ A reference to the Messenger can be sent via an Intent using the previously ment
 
 Messages sent by the remote process via the messenger are delivered to the local handler. Great for efficient call-backs from the service to the client
 
+#### Permissions
+Because Android applications are installed in a sandbox and initially it does not have access to neither user information nor access to system components (such as using the camera or the microphone), it provides a system based on permissions where the system has a predefined set of permissions for certain tasks that the application can request.
+As an example, if you want your application to use the camera on the phone you have to request the camera permission. 
+On Android versions before Marshmallow (API 23) all permissions requested by an application were granted at installation time. From Android Marshmallow onwards the user have to approve some permissions during application execution.
+
+##### Protection Levels
+Android permissions are classified in four different categories based on the protection level it offers.
+- *Normal*: Is the lower level of protection, it gives applications access to isolated application-level feature, with minimal risk to other applications, the user or the system. It is granted at installation. It is the default value.

```

Comment:
Maybe you say "They are granted during installation of the App."

What do you mean with "It is the default value."?